### PR TITLE
Adding arrow-next needed by the TaskManagement Reporting page

### PR DESCRIPTION
**Pull request by kyythane**

### CODE REVIEW

File: docs/styles/_icons-section.scss [hunk 1]
```diff
@@ -89,6 +89,10 @@
     @include icon('arrow-back');
   }
 
+  .icon-arrow-next {
+    @include icon('arrow-next');
+  }
```
Comment (SamHerbert) [inline docs/styles/_icons-section.scss]: Want to add this to the docs/examples, too?

**Comment by nhinsch** [comment]
Don't forget to increment the version of Juice in the `package.json` file! 🙂

File: docs/styles/_icons-section.scss (hunk 1 above)
Comment (kyythane) [reply docs/styles/_icons-section.scss]: Oh yes. Missed that

**Comment by kyythane** [comment]
Updated PR

**Pull request merged**